api/src/indigo_internal.h

```diff
@@ -159,6 +160,10 @@ class DLLEXPORT IndigoObject
         TGROUP,
         TGROUPS_ITER,
         GROSS_REACTION,
+        JSON_LOADER,
+        JSON_SAVER,

```

Comment:
Do we need JSON_LOADER and JSON_SAVER if the corresponding classes are not used?